Running cluster for deep fits
120021 120022 normal
Running object 120021
WARNING: Could not memory map array with mode='readonly', falling back to mode='denywrite', which means that the array will be read-only [astropy.io.fits.file]
Min gap is: 0.0
New time_gap is :0.004
Number of observations now:  33
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  63
Min gap is: 0.00390625
New time_gap is :0.00790625
Number of observations now:  27
Not enough z observations!
Counts: {'g': 33, 'r': 62, 'i': 26}
Mu: {'g': 0.17109123, 'r': 0.17732477, 'i': 0.22793001, 'z': 0.0}
No handles with labels found to put in legend.
Variable??
0.022815559609602132
1.1041854306055385
Non-Variable source in g
Variable??
0.01992004962824389
1.248116372735886
Non-Variable source in r
Variable??
0.03877733073643945
1.0084466835840051
Non-Variable source in i
Variable??
0
0
Non-Variable source in z
Variable:    0/4
Var Bands:   
Slope:       [0.445819215769677, 1, 1, 1]
Slope Error: [0.19615379254717447, 0, 0.23699695706604146, 0.0]
intercepts:  [0.049419957933983794, 0, -0.09485092222510612, 0]
Int Error:   [0.03553620730236384, 0, 0.03302446058261102, 0.0]
StDev:       [0.040345483, 0.047078565, 0.046336923, 1.9246581]
Not variable, closing plots & going to next row
Successfully finished!
Rows 120021 to 120022 ran in 2.5522 seconds
